### ISSUE TITLE
Evaluate rank seems wrong

### ISSUE THREAD

**Issue by moghammed** [zchYellow/TransE#11]
In the `evaluate` method of model.py, the rank calculation seems wrong to me. 

If I understand correctly, `tf.nn.top_k(tf.reduce_sum(tf.abs(distance_head_prediction), axis=1), k=self.kg.n_entity)` will return the entities that are furthest from the predicted head, instead of the closest. So shouldn't this line be

`tf.nn.top_k(-tf.reduce_sum(tf.abs(distance_head_prediction), axis=1), k=self.kg.n_entity)`

**Comment by ZichaoHuang** [comment]
I do the trick at [line 235](https://github.com/ZichaoHuang/TransE/blob/master/src/model.py#L235) and [line 244](https://github.com/ZichaoHuang/TransE/blob/master/src/model.py#L244) of model.py.

**Comment by moghammed** [comment]
Ah, yes, I see. Thanks for the clarification!
